docs/getting_started/running.md

```diff
@@ -10,7 +10,7 @@ irodsServer -d
 
 The server will perform various checks during startup to make sure it is safe to accept client requests. The most important check is validation of `server_config.json`. Failing validation will result in the server terminating immediately. All validation errors are reported via stderr.
 
-Logs are written to syslog. See [The Server Log](../../system_overview/troubleshooting/#the-server-log) for information regarding syslog.
+Logs are written to syslog. See [The Server Log](../../system_overview/server_log/#the-server-log) for information regarding syslog.
```

Comment:
can probably remove the anchor at the end...

```suggestion
Logs are written to syslog. See [The Server Log](../../system_overview/server_log/) for information regarding syslog.
```

Reply:
We can. I left it in as a future proofing measure - i.e. sections get shuffled around later on.

Reply:
if it's intentional, leave it.   it works.